roles/base/templates/docker_facts.j2

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+dver=$(sudo docker version | awk '/Version/{print $2; exit 0}')
+exper=$(sudo docker version | awk '/Experiment/{print $2; exit 0}')
```

Comment:
With some changes to cover docker not being present etc., this can be made to work. But, how is it better than what I implemented? I prefer my method because it is easier to extend.